app/views/components/_attachment_meta.html.erb

```diff
@@ -0,0 +1,42 @@
+<%
+  attachment = GovukPublishingComponents::Presenters::Attachment.new(attachment)
+  attributes = []
+
+  if attachment.content_type_name
```

Comment:
Slightly outside of the scope of this PR: could we extract this into the presenter in the main components gem, so we could simplify the component and the code here?

All it seems to do is defined a bunch of textual attributes (the spans/classes don't add anything, individually), and then join them together with a comma [later on](https://github.com/alphagov/content-publisher/pull/1984/commits/c9bf7e60b561acf6de76acaa0baa7f8755a95ccf#diff-bb4f2ef4fecbbfab06aea98c17b6d0b3R40).

Reply:
Hi - you're correct except for the [`gem-c-attachment__abbr` class](https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_attachment.html.erb#L11-L13), which does have some associated styling & behaviour. If we didn't have to worry about this, we could indeed move this out to the presenter and just call `attachment.attributes` later on.

It would still be possible to refactor by, say, returning an array of hashes of the form `{ text: "foo", class: "gem-c-attachment__abbr", title: "Expanded foo" }`, where the majority of hashes would just contain a `text` key/value pair. It means the view needs to know what to do with those hashes, though. I can kind of see why it was just left in the view now.